lib/live_debugger/live_views/live_views_dashboard_live.ex

```diff
@@ -94,41 +66,111 @@ defmodule LiveDebugger.LiveViews.LiveViewsDashboardLive do
   end
 
   @impl true
-  def handle_event(
-        "lv-process-picked",
-        %{"socket-id" => socket_id, "transport-pid" => transport_pid},
-        socket
-      ) do
+  def handle_event("refresh", _params, socket) do
     socket
-    |> push_navigate(to: Routes.channel_dashboard(socket_id, transport_pid))
+    |> assign(:groupped_lv_processes, AsyncResult.loading())
+    |> assign_async_groupped_lv_processes()
     |> noreply()
   end
 
-  @impl true
-  def handle_event("refresh", _params, socket) do
-    socket
-    |> assign(:lv_processes, AsyncResult.loading())
-    |> assign_async_lv_processes()
-    |> noreply()
+  attr(:transport_pid, :any, required: true)
+  attr(:lv_processes, :list, required: true)
+
+  defp tab_group(assigns) do
+    assigns = assign(assigns, :lv_processes_length, length(assigns.lv_processes))
+
+    ~H"""
+    <div class="w-full h-max flex flex-col shadow-custom rounded-sm bg-surface-2-bg border border-default-border">
+      <div class="pl-4 p-3 flex items-center h-10 border-b border-default-border">
+        <p class="text-primary-text text-xs font-medium">
+          <%= Parsers.pid_to_string(@transport_pid) %>
+        </p>
+      </div>
+      <div class="w-full flex bg-surface-0-bg">
+        <.list elements={Enum.with_index(@lv_processes)}>
+          <:item :let={{lv_process, index}}>
+            <div class="flex items-center w-full">
+              <.nested_indent :if={lv_process.nested?} last?={index == @lv_processes_length} />
+              <.list_element lv_process={lv_process} />
+            </div>
+          </:item>
+        </.list>
+      </div>
+    </div>
+    """
   end
 
-  defp event_values_map(lv_process) do
-    %{
-      "phx-value-socket-id" => lv_process.socket_id,
-      "phx-value-transport-pid" => Parsers.pid_to_string(lv_process.transport_pid)
-    }
+  attr(:last?, :boolean, required: true)
+
+  defp nested_indent(assigns) do
+    ~H"""
+    <div class="relative w-8 h-12">
+      <div class={[
+        "absolute top-0 right-2 w-1/4 h-1/2 border-b border-default-border",
+        if(not @last?, do: "border-l")
+      ]}>
+      </div>
+      <div :if={@last?} class="absolute top-0 left-2 w-1/4 h-full border-r border-default-border">
```

Comment:
I think you can try using tailwing `last:` prefix 
https://v3.tailwindcss.com/docs/hover-focus-and-other-states#first-last-odd-and-even